Sources/FuzzilliCli/main.swift

```diff
@@ -317,8 +329,19 @@ for (generator, var weight) in (additionalCodeGenerators + regularCodeGenerators
 //
 
 func makeFuzzer(for profile: Profile, with configuration: Configuration) -> Fuzzer {
+    let differentialTesting = isNetworkMaster || configuration.differentialRate > 0.0 || configuration.differentialWeaveRate > 0.0
```

Comment:
Why the `isNetworkMaster` flag?

Reply:
If an instance receives differential samples via the network we need the reference runner to determine whether they're flaky or deterministic.

Reply:
I think I'd prefer to just log an error in that case and save the sample as flaky crash without trying to reproduce it. I think that should be fairly easy to do? This way we avoid spinning up unused processes for non-differential fuzzing sessions